client/components/app.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import axios from 'axios';
+import PartySize from './partySize.jsx';
+import Date from './date.jsx';
+import Time from './time.jsx';
+
+class App extends React.Component { 
+	constructor(props) {
+		super(props);
+		this.state = {
+			restaurantId: '',
+		}
+	} 
+	getInfo () {
+		axios.get('/restaurant/:restaurant_id')
```

Comment:
The use of the colon to indicate parameter in the endpoint "/restaurant/:restaurant_id" only works in express as part of the express.router functionality. App will need to send explicit requests to the server.